azurelinuxagent/common/dhcp.py

```diff
@@ -67,7 +67,7 @@ def wait_for_network(self):
         Wait for network stack to be initialized.
         """
         ipv4 = self.osutil.get_ip4_addr()
-        while ipv4 == '' or ipv4 == '0.0.0.0':  # pylint: disable=R1714
+        while ipv4 == '' or ipv4 == '0.0.0.0':
```

Comment:
I personally prefer using the `in` that this warning (`R1714=consider-using-in`) suggests, e.g. `ipv4 in ('', '0.0.0.0')`, but I don't think it really matters here. I'm fine with it as is, but figured I should mention it because I've been pointing out the `R17..` suppressions.

Reply:
Feel free to fix those on a separate PR